File: app/src/main/java/com/tekzee/amiggoss/base/model/NotificationData.java
package com.tekzee.amiggoss.base.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class NotificationData {

    @SerializedName("notification")
    @Expose
    private Notification notification;
    @SerializedName("data")
    @Expose
    private Data data;
    @SerializedName("to")
    @Expose
    private String to;

    public Notification getNotification() {
        return notification;
    }

    public void setNotification(Notification notification) {
        this.notification = notification;
    }

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    public String getTo() {
        return to;
    }

    public void setTo(String to) {
        this.to = to;
    }


    public NotificationData(Notification notification, Data data, String to) {
        this.notification = notification;
        this.data = data;
        this.to = to;
    }
}

File: app/src/main/java/com/tekzee/amiggoss/ui/notification/model/NotificationData.kt
package com.tekzee.amiggoss.ui.notification.model


import com.google.gson.annotations.SerializedName

data class NotificationData(
    var loadingStatus: Boolean = false,
    @SerializedName("data")
    val `data`: String = "",
    @SerializedName("device_type")
    val deviceType: String = "",
    @SerializedName("id")
    val id: Int = 0,
    @SerializedName("is_read")
    val isRead: Int = 0,
    @SerializedName("message")
    val message: String = "",
    @SerializedName("notification_key")
    val notificationKey: Int = 0,
    @SerializedName("subject")
    val subject: String = ""
)